packages/tocco-ui/src/Popover/StyledPopover.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components'
+
+import {declareTypograhpy} from '../Typography'
+import {theme} from 'styled-system'
+
+const borderWidth = 1
```

Comment:
constants like this could be written with constant-case:
BORDER_WIDTH

its a little bit hard to draw the line there since almost every var is a const but i usually use constant-case for setting const